docs/static/releasenotes.asciidoc

```diff
@@ -1,4 +1,94 @@
 [[releasenotes]]
 == Release Notes
-coming[9.0.0]
 
+* <<logstash-9-0-0-beta1,Logstash 9.0.0-beta1>>
+
+[[logstash-9-0-0-beta1]]
+=== Logstash 9.0.0-beta1 Release Notes
+
+==== Breaking changes
+* Default buffer type to 'heap' for 9.0 https://github.com/elastic/logstash/pull/16500[#16500]
+* Remove `http.*` settings https://github.com/elastic/logstash/pull/16552[#16552]
+* Remove the Arcsight module and the modules framework https://github.com/elastic/logstash/pull/16794[#16794]
+* Remove `event_api.tags.illegal` for 9.0 https://github.com/elastic/logstash/pull/16461[#16461]
+* Remove deprecated modules netflow, fb_apache and azure https://github.com/elastic/logstash/pull/16514[#16514]
+* Remove ingest-converter https://github.com/elastic/logstash/pull/16453[#16453]
+* Drop JDK 11 support https://github.com/elastic/logstash/pull/16443[#16443]
+* Remove enterprise search from default distribution https://github.com/elastic/logstash/pull/16818[#16818]
+* Enable `log.format.json.fix_duplicate_message_fields` by default https://github.com/elastic/logstash/pull/16578[#16578]
+
+==== New features and enhancements
+* Add `%{{TIME_NOW}}` pattern for sprintf https://github.com/elastic/logstash/pull/16906[#16906]
```

Comment:
This one is interesting...

It is new to `9.0.0-beta1`, but by the time `9.0.0` comes out, it will also be on `8.18.0`. 

I'm inclined to remove it - what do you think @karenzone?

Reply:
I have removed this as this will be in 8.18 and not specific to 9.0. I will put back if there is any strong opinion.

Reply:
FYI: removing this item lead New features become empty, so removed entire "New features and enhancements" section.